maestro-scalding/src/test/scala/au/com/cba/omnia/maestro/task/SqoopExportExecutionSpec.scala

```
@@ -102,9 +104,10 @@ object SqoopExportExecutionSpec
       val config = SqoopExportConfig(
         options(table).sqlQuery(s"select * from $table"), deleteFromTable = true
       )
-      execute(sqoopExport(config, exportDir)) must throwA[RuntimeException](
-        message = "SqoopOptions.getSqlQuery must be empty on Sqoop Export with delete from table"
-      )
+      execute(sqoopExport(config, exportDir)) must beLike {
+        case Failure(ex: RuntimeException) =>
+          ex.getMessage mustEqual "SqoopOptions.getSqlQuery must be empty on Sqoop Export with delete from table"
+      }
```

Now that `SqoopEx.sqoopExport` is embedded in a flatMap, seems that errors are caught an embedded in the `Execution`, rather than getting thrown.

Technically this is a breaking change, however it is also our preferred error handling strategy. Also, our users would already be performing error handling at the monad-level, so it's not like this error would get swallowed after this change.

---

Agreed.